app/services/force/base.rb

```diff
@@ -28,6 +28,10 @@ def cache_query(q)
 
     # run a Salesforce SOQL query
     def query(q)
+      if Rails.env.development?
+        puts "SOQL>"
+        puts q
+      end
```

Comment:
We have it in our our feature branchs and this is already in master. this is going to be merged into lap_v1_master. I would leave it since it's very useful.